datumaro/plugins/data_formats/video.py

```diff
@@ -124,3 +127,87 @@ def __iter__(self):
     def get(self, id, subset=None):
         assert subset == self._subset, "%s != %s" % (subset, self._subset)
         return super().get(id, subset or self._subset)
+
+
+class VideoKeyframesImporter(VideoFramesImporter):
+    """
+    Reads video frames as a dataset.
+    """
+
+    @classmethod
+    def build_cmdline_parser(cls, **kwargs):
+        parser = super().build_cmdline_parser(**kwargs)
+        parser.add_argument(
+            "-t", "--threshold", type=float, default=0.3, help="Frame step (default: %(default)s)"
+        )
+        return parser
+
+    @classmethod
+    def find_sources(cls, path):
+        if not osp.isfile(path):
+            return []
+        return [{"url": path, "format": VideoKeyframesBase.NAME}]
+
+
+_FEAT_SIZE = (64, 64)
```

Comment:
Can it be configurable?

Reply:
I added a new parameter for adjusting the image size during the ZNCC computation.